pkg/ddevapp/ddevapp.go

```diff
@@ -388,6 +390,22 @@ func (app DdevApp) GetDocroot() string {
 	return app.Docroot
 }
 
+// GetAppRoot return the full path from root to the app directory
+func (app *DdevApp) GetRelativeWorkingDirectory() (string, error) {
+	re, err := regexp.Compile(app.AppRoot + `/`)
```

Comment:
I don't think regex is the right thing to do here. The AppRoot may have regex characters in it, like a "." for example, which could match anything.

How about strings.TrimPrefix() - seems easier.